testing/razercfg/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Taner Tas <taner76@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=razercfg
+pkgver=0.40
+pkgrel=0
+pkgdesc="Razer mouse configuration tool."
+url="https://bues.ch/cms/hacking/razercfg.html"
+arch="all"
+license="GPL-2.0-or-later"
+makedepends="
+	cmake
+	libusb-dev
+	python3
+	"
+subpackages="$pkgname-openrc $pkgname-gui::noarch"
+source="
+	https://github.com/mbuesch/razer/archive/$pkgname-$pkgver.tar.gz
+	razerd.initd
+	"
+options="!check" # No test suite
+builddir="$srcdir/razer-$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	cmake . -DCMAKE_INSTALL_PREFIX='/usr'
```

Comment:
You are not running make here?
I think make install is not triggering it which is not the correct location.

Reply:
I'm not sure what is your confusion bu I have to say that his is the cmake equivalent of --prefix=/usr.

Reply:
Maybe I can explain a little more:

The README says that both cmake and make are used for the build step. cmake creates the Makefile. make uses the Makefile to build the binaries. https://github.com/mbuesch/razer#building

make install is correctly called in the package step. If the binary files are not available, so if make has not been called in the build step, make will notice and compile them. Compilation should not happen in the package step, so we're suggesting you add a make call to build().